services/core-api/app/api/mines/explosives_permit_amendment/resources/explosives_permit_amendment.py

```diff
@@ -14,6 +14,18 @@
 from app.api.mines.explosives_permit_amendment.models.explosives_permit_amendment import ExplosivesPermitAmendment
 class ExplosivesPermitAmendmentResource(Resource, UserMixin):
     parser = CustomReqparser()
+    parser.add_argument(
+        'amendment',
+        type=str,
+        store_missing=False,
+        required=False,
```

Comment:
I was wondering what this referred to! Can we add a help="some text to describe what's expected here"? I am not too keen on doing the string formatting on the FE for a value used on the BE, would prefer to send just the amendment number and do the string formatting where it's being used. 
The amendment_with_date though looks like it should be calculated on the BE if we're only using it on that letter template. Is it used anywhere on the FE?